src/version.h

```diff
@@ -11,7 +11,7 @@
  */
 
 
-static const int PROTOCOL_VERSION = 70221;
+static const int PROTOCOL_VERSION = 70222;
```

Comment:
Does `MIN_MASTERNODE_PROTO_VERSION` need to match this since we're adding a new quorum? Also, should `ISDLOCK_PROTO_VERSION` be changed also (I think there were changes to it since 70220)?

Reply:
@PastaPastaPasta @UdjinM6 ?

Reply:
Yeah, we probably should to ban old MNs quicker once enough people upgrade. Although I don't think it's strictly needed